.github/scripts/github_metrics.py

```diff
@@ -0,0 +1,91 @@
+"""
+Submit metrics returns "Payload accepted" response
+"""
+
```

Comment:
For reference: https://docs.datadoghq.com/api/latest/metrics/

Reply:
I see there's a new secret that we would be adding `secrets.DATADOG_API_KEY`, any other required changes in existing permissions?

Reply:
Nope, just this key